Source/WebCore/platform/ios/WebEvent.h

```diff
@@ -108,6 +108,18 @@ typedef enum {
 } WebKeyboardInputFlagValues;
 typedef NSUInteger WebKeyboardInputFlags;
 
+#ifndef WK_HAVE_UI_KEY_EVENT
```

Comment:
WK_HAVE??

Reply:
Ah, yeah — I wasn't sure about this :/

Maybe it would be better to just write the `#if __has_include` out in these three places?